File: FurnitureFront/src/app/models/product-edition.ts
import { Product } from 'src/app/models/product';
import { Edition } from 'src/app/models/edition';
export interface ProductEdition {
    id: number;
    editionId: number;
    productId: number;
    priceMod: number;
    edition: Edition;
    product: Product;
}

File: FurnitureBackend/Webshop.Domain/ProductEdition.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Webshop.Domain
{
    public class ProductEdition
    {
        public int Id_PE { get; set; }
        public int VersionId { get; set; }
        public int ProductId { get; set; }
        public int PictureId { get; set; }
        public Edition Editions { get; set; }
        public Product Products { get; set; }
        public List<Picture> Pictures = new List<Picture>();
    }
}
